### PR TITLE
New nldi api image

### PR DESCRIPTION
**Pull request by mwernimont**
Before making a pull request
----------------------------

- [x] Update the changelog appropriately

New Home page projects
-----------
4 new projects needed to be added to the home page.  GDP, Geoserver, Cloud Optimized Data Access and Pygeoapi respectively.

Description
-----------
https://code.usgs.gov/wma/nhgf/nhgf/-/issues/3

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: content/coda.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+date: 2022-05-10
+slug: #
```
Comment (EthanGrahn) [inline content/coda.md:3]: I'm not familiar with how the parsing is handled in this situation. Will the slug value be inserted as an empty string, or does it need to be explicitly defined as `""`?

**Comment by EthanGrahn** [comment]
I have one question mentioned above, otherwise this pull request looks good to me.

File: content/coda.md (hunk 1 above)
Comment (mwernimont) [reply content/coda.md:3]: So far as I could tell the slug was for generating links inside the site, so I used the # as I do when I have a link I don't currently want to go anywhere.  When I tested this it did as I expected and created a link just to localhost:1313.  I do not believe the idea is to keep it linked to nothing, but the decision hasn't currently been made where we want this one to link to.

Comment (brownd-usgs) [reply content/coda.md:3]: Yeah, still deciding where we want it to link to. Let me know if there's no good way to handle this situation and I'll give you somewhere to link it to.

**Comment by mwernimont** [comment]
@jkreft-usgs looks like you're the only one out of the 4 of us with merge powers.  Mind giving this a look when you have time and merging if you think it looks alright?

File: content/coda.md (hunk 1 above)
Comment (jkreft-usgs) [reply content/coda.md:3]: there needs to be a string.  The "slug" is the URL that this page would be on

Comment (mwernimont) [reply content/coda.md:3]: @brownd-usgs got a URL for now to use then?

**Comment by mwernimont** [comment]
@jkreft-usgs Tile with the stringless slug removed.

**Comment by jkreft-usgs** [comment]
confirming that this actually renders locally?

**Comment by mwernimont** [comment]
I have a local development server yes

**Comment by mwernimont** [comment]
Screen Shot 2022-05-16 at 12 21 09 PM

**Pull request merged**